example/styleguide/bundle.js

```diff
@@ -1,10 +1,26 @@
 define([
     "./src/ExampleStyleGuideModelProvider",
     "./src/MCTExample",
+    "text!./res/templates/intro.html",
```

Comment:
`text!` still necessary here?

Reply:
Yes but we can do even better with html-loader.  Updated.